blender/arm/material/make_mesh.py

```diff
@@ -536,15 +542,25 @@ def make_forward(con_mesh):
                 frag.add_uniform('sampler2DShadow shadowMapSpot[4]', included=True)
 
     if not blend:
-        mrt = rpdat.rp_ssr  # mrt: multiple render targets
```

Comment:
Why remove the comment that explains this abbreviation? I'm not sure how common it is, but I would imagine that the average reader doesn't know what "mrt" is supposed to say.